plugins/algoltek-usb/fu-algoltek-usb-device.c

```diff
@@ -91,11 +93,30 @@ fu_algoltek_usb_device_rdv(FuAlgoltekUsbDevice *self, GError **error)
 						  error))
 		return NULL;
 
-	/* Remove len and cmd bytes */
-	for (guint32 i = 2; i < st->len; i++) {
-		if (st->data[i] < 128)
-			fu_byte_array_append_uint8(version_data, st->data[i]);
+	if (!fu_memread_uint16_safe(st->data, st->len, 2, &version_prefix, G_BIG_ENDIAN, error))
+		return NULL;
+
+	if (version_prefix == 0x4147) {
+		/* Remove len, cmd bytes and "AG" prefixes*/
```

Comment:
little case `remove` please -- and a space before the `*/`

Reply:
Updated! Sorry for this.